omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestController1_10Test.java

```diff
@@ -56,6 +73,9 @@
 	@Autowired
 	private AttachmentsContext attachmentsContext;
 	
+	@Autowired
+	ComplexDataHelper1_10 complexDataHelper;
```

Comment:
This should go away, just get the injected bean through `attachmentsContext`, that you can rename to `ctx` here again for the sake of concision.

Reply:
i have fixed it